subsys/rpmsg_service/Kconfig

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2020-2021 Nordic Semiconductor (ASA)
+# SPDX-License-Identifier: Apache-2.0
+
+# Workaround for not being able to have commas in macro arguments
+DT_CHOSEN_Z_IPC_SHM := zephyr,ipc_shm
+DT_CHOSEN_Z_IPC := zephyr,ipc
+DT_CHOSEN_Z_IPC_TX := zephyr,ipc_tx
+DT_CHOSEN_Z_IPC_RX := zephyr,ipc_rx
+
+config RPMSG_SERVICE_SINGLE_DEV_SUPPORT
+	bool
+	default $(dt_chosen_enabled,$(DT_CHOSEN_Z_IPC))
```

Comment:
Basically OpenAMP can be configured in two ways:

1. Single IPM is used for TX and RX
2. Two separate IPMs are used for TX and RX

Perhaps if I renamed this option it would be clearer. How about `RPMSG_SERVICE_SINGLE_IPM_SUPPORT`?

This option is not shown in menuconfig. Is it OK to add help string to such options?

Reply:
> Basically OpenAMP can be configured in two ways:
> 
> 1. Single IPM is used for TX and RX
> 2. Two separate IPMs are used for TX and RX
> 
> Perhaps if I renamed this option it would be clearer. How about `RPMSG_SERVICE_SINGLE_IPM_SUPPORT`?
Ok for me
> 
> This option is not shown in menuconfig. Is it OK to add help string to such options?
If that doesn't generate error, i would say yes :).